tests/lib/rook/ses.py

```diff
@@ -47,6 +81,11 @@ def preinstall(self):
         self._fix_yaml()
 
     def _get_rook(self):
+        # do not use rpm-package for self-built rook images
+        if converter('@bool', settings.UPSTREAM_ROOK.BUILD_ROOK_FROM_GIT):
```

Comment:
In this case the context is SES. So if we're building as part of this test we should perhaps use a different config (that defaults to github.com/SUSE/rook)

Reply:
IMO not. The context for rook is still upstream. The context for ceph is SES.
What do you think?

Reply:
Upstream care about ceph as well. This file (`ses.py`) is specifically for deploying the ses version of rook-ceph. I think we do want to use a different variable here.

Reply:
I am fine with that. Will change it.